torch/_C/__init__.pyi.in

```diff
@@ -1513,9 +1514,16 @@ class _DispatchModule:
         dispatch: str,
         debug: str = "impl_t_t",
     ) -> _DispatchModule: ...
-    def impl(self, name: str, dispatch: str, func: Callable) -> _DispatchModule: ...
-    def define(self, schema: str, alias: str = "") -> _DispatchModule: ...
+    def impl_with_aoti_compile(
+        self,
+        ns: str,
+        op_name_with_overload: str,
+        dispatch: str
+    ) -> None: ...
+    def impl(self, name: str, dispatch: str, func: Callable) -> None: ...
+    def define(self, schema: str, alias: str = "") -> str: ...
     def fallback_fallthrough(self, dispatch: str = "") -> _DispatchModule: ...
+    def fallback(self, dispatch: str, func: Callable, with_keyset: _bool = False) -> None: ...
```

Comment:
`dispatch` is of the `_dispatchkey` type, I guess

https://github.com/pytorch/pytorch/blob/b0bafd2be51d326acff7130994976280d2810584/torch/_C/__init__.pyi.in#L1538-L1541